src/router.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const express = require('express');
+const db = require('./db');
+
+const router = express.Router();
+
+
+// register
+router.post('/register', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+// login
+
+router.post('/login', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+// Gets all items
+router.get('/allproducts', (req, res, next) => {
```

Comment:
Change the route to /product

Reply:
I changed my routes to /clothing as i realized that would make more sense since this is referring to a particular route.